middle_end/flambda2/docs/types.md

```diff
@@ -433,14 +433,14 @@ make the whole environment Bottom (if one of the non-existential variables ends
 up having this special value or being related to one of them, then the whole
 environment becomes Bottom). This special kind of value is sometimes called
 Poison in other settings.
-In practice this means that existential variables from one side only are
```

Comment:
This one is not a typo. But if it appears as one, then maybe moving "only" before "one side" would make the sentence easier to read.

Reply:
OK. I've reverted this change. I think it reads fine as it was.